### PR TITLE
guard against integer overflow

### PR DESCRIPTION
**Pull request by djmitche**
This should silence a CodeQL warning that appeared out of nowhere today.

### CODE REVIEW

**Comment by djmitche** [comment]
Worth noting this was not a security issue -- the things being merged are all controlled by the deployer

File: tools/worker-runner/cfg/workerconfig.go [hunk 1]
```diff
@@ -102,6 +102,10 @@ func merge(v1, v2 interface{}) interface{} {
 	arr1, arr1ok := v1.([]interface{})
 	arr2, arr2ok := v2.([]interface{})
 	if arr1ok && arr2ok {
+		// check for overflw
```
Comment (jwhitlock) [inline tools/worker-runner/cfg/workerconfig.go:105]: nit
```suggestion
		// check for overflow
```

File: tools/worker-runner/cfg/workerconfig.go [hunk 2]
```diff
@@ -102,6 +102,10 @@ func merge(v1, v2 interface{}) interface{} {
...
+		if len(arr1) > 2*1024*1024*1024 || len(arr2) > 2*1024*1024*1024 {
```
Comment (jwhitlock) [inline tools/worker-runner/cfg/workerconfig.go:106]: Glad we still support merging 1 billion item lists, I'm sure the system will have no problems at that scale

**Review by jwhitlock — APPROVED** [review]
Thanks @djmitche!

File: tools/worker-runner/cfg/workerconfig.go (hunk 1 above)
Comment (djmitche) [reply tools/worker-runner/cfg/workerconfig.go:105]: I feel like at this point I should claim that I leave these here for you intentionally

**Pull request merged**